docs/docs/guide/4-submitting-image-queries.md

```diff
@@ -0,0 +1,150 @@
+# Submitting Image Queries
+Once you have [created a `Detector`](./3-working-with-detectors.md) and [captured an image](./2-grabbing-images.md), you can submit your image to Groundlight for analysis.
+
+## Submit an Image Query
+The primary method for submitting an image query is `submit_image_query(detector: Detector, image: Any)`. This method takes a `Detector` object and an image as input and returns an `ImageQuery` object.
+```python notest
+from groundlight import Groundlight
+
+gl = Groundlight()
+detector = gl.get_detector(id="det_abcdef...")
+
+# highlight-start
+image_query = gl.submit_image_query(detector=detector, image="path/to/image.jpg")
+# highlight-end
+```
+
+`submit_image_query` provides fine-grained control over how the `ImageQuery` is processed. For example, a per-query confidence threshold can be set (defaults to the `Detector`'s confidence threshold), and the query can be set to wait for up to a certain amount of time for a confident response (defaults to 30s). For example:
+```python notest
+from groundlight import Groundlight
+
+gl = Groundlight()
+detector = gl.get_detector(id="det_abcdef...")
+
+# highlight-start
+image_query = gl.submit_image_query(
+    detector=detector,
+    image="path/to/image.jpg",
+    confidence_threshold=0.95,
+    wait=10.0,  # seconds
+)
+# highlight-end
+```
+
+See the [API Reference](../api-reference/api-reference.md) for more information on the `submit_image_query` method.
+
+## Aliases for `submit_image_query`
+For convenience, the `submit_image_query` method has aliases for the different patterns of usage. These aliases are `ask_confident`, `ask_ml`, and `ask_async`.
+
+### Get the first confident answer
+`ask_confident` evaluates an image with Groundlight waiting until an answer above the confidence threshold
+        of the detector is reached or the wait period has passed.
+```python notest
+from groundlight import Groundlight
+
+gl = Groundlight()
+detector = gl.get_detector(id="det_abcdef...")
+
+# highlight-start
+image_query = gl.ask_confident(detector=detector, image="path/to/image.jpg")
+# highlight-end
+```
+
+### Get the first available answer, regardless of confidence
+`ask_ml` evaluates an image with Groundlight and returns the first answer Groundlight can provide, agnostic of confidence. There is no wait period when using this method. It is called `ask_ml` because our machine learning models are earliest on our escalation ladder and thus always the fastest to respond.
```

Comment:
```suggestion
`ask_ml` evaluates an image with Groundlight and returns the first answer Groundlight can provide, agnostic of confidence. There is no wait period when using this method. It is called `ask_ml` because our machine learning models are earliest on our escalation ladder and thus always the fastest to respond. *NOTE* We recommend using the `ask_confident` or the `ask_async` methods whenever possible for best results. 
```